cmd/algod/main.go

```diff
@@ -56,6 +56,7 @@ var listenIP = flag.String("l", "", "Override config.EndpointAddress (REST liste
 var sessionGUID = flag.String("s", "", "Telemetry Session GUID to use")
 var telemetryOverride = flag.String("t", "", `Override telemetry setting if supported (Use "true", "false", "0" or "1")`)
 var seed = flag.String("seed", "", "input to math/rand.Seed()")
+var stopAtRound = flag.String("round", "", "Stop catchup at this block number")
```

Comment:
The flag library can convert this number automatically
```suggestion
var stopAtRound = flag.Uint64("round", "", "Stop catchup at this block number")
```

Reply:
`Stop` has been renamed to `Pause` and resume functionality is added to catchup service